### ISSUE TITLE
Typo instantiating loss at zero

### ISSUE THREAD

**Issue by aymebou** [pytorch/tutorials#844]
https://github.com/pytorch/tutorials/blob/48e5cccecbd385e00b95cb60ecdfecd06a43282b/intermediate_source/char_rnn_generation_tutorial.py#L281

Hello folks at Pytorch,

When instantiating loss at a real zero, i think you rather meant something like (i don't know if there is a better way but I usually do it this way) : 

`loss = torch.autograd.Variable(torch.Tensor([0]))`

**Comment by zabboud** [comment]
I would take this on but first I wanted some clarity on the question - is the question whether setting `loss = 0` vs. `loss = torch.autograd.Variable(torch.Tensor([0]))` is better? Or just to explain why `loss = 0` is used instead?

**Comment by abhi-glitchhg** [comment]
>When instantiating loss at a real zero, i think you rather meant something like (i don't know if there is a better way, but I usually do it this way) :

I think instantiating the loss with 0 (or python `int` dtype) is not a problem because once you use the `add` operation with an integer and a tensor 

https://github.com/pytorch/tutorials/blob/48e5cccecbd385e00b95cb60ecdfecd06a43282b/intermediate_source/char_rnn_generation_tutorial.py#L286

the dtype is promoted to tensor[[1](https://pytorch.org/docs/stable/tensor_attributes.html#type-promotion-doc)][[2](https://pytorch.org/docs/stable/generated/torch.add.html)].   

But initializing it as a tensor won't do any harm and could avoid future confusion like this.

**Comment by akjalok** [comment]
\assigntome
